project/resources/tests.py

```diff
@@ -1,88 +1,110 @@
 
 
     def test_view_resources(self):
+        records = randint(2, 102)
+        create_batch(ResourceFactory, records)
+
         url = '/api/v1/resources/'
         response = self.client.get(url)
         self.assertEqual(response.status_code, status.HTTP_200_OK)
-        self.assertEqual(response.data['count'], 31)
+        self.assertEqual(response.data['count'], records )
 
 
     def test_search_resources(self):
+        create(ResourceFactory, title="Elm Crash-Course (Reactivate London)" )
+        create(ResourceFactory, title="elm Documentation")
+        create(ResourceFactory, description="The best place to start is the official guide. It will give you a solid foundation for creating applications with Elm. Once you have worked through that, the next place to look for documentation is on the packages you are using.")
+        create(ResourceFactory, description="Elm is a programming language targeted at the front end (runs in a browser, not on Node) that offers a different take on building dynamic web applications.")
+
         url = '/api/v1/resources/?search=Elm'
         response = self.client.get(url)
         self.assertEqual(response.status_code, status.HTTP_200_OK)
-        self.assertEqual(response.data['count'], 3)
+        self.assertEqual(response.data['count'], 4)
         self.assertContains(response, "Elm Crash-Course (Reactivate London)")
+        self.assertContains(response, "or creating applications with Elm. ")
 
 
     def test_view_one_resource(self):
+        new_resource = create(ResourceFactory, guid='96f1ee80-59f4-11ea-9149-dca9047779fe')
         url = '/api/v1/resources/96f1ee80-59f4-11ea-9149-dca9047779fe/'
+
         response = self.client.get(url)
         self.assertEqual(response.status_code, status.HTTP_200_OK)
-        self.assertEqual(response.data['title'],"Linear Digressions")
-        self.assertEqual(response.data['description'], "Linear Digressions is a podcast about Machine Learning and Data Science")
+        self.assertEqual(response.data['title'], new_resource.title)
+        self.assertEqual(response.data['description'], new_resource.description)
 
 
     def test_patch_one_resource(self):
-        url = '/api/v1/resources/96f5e4ea-59f4-11ea-9149-dca9047779fe/'
+        new_resource = create(ResourceFactory, guid='96f1a740-59f4-11ea-9149-dca9047779fe')
+        url = f'/api/v1/resources/{new_resource.guid}/'
 
         data= {
             "description": "A _diabolically irresponsible_ talk in which I celebrate modern Python coding by **abandoning all backwards compatibility** THE END",
-            "tags": ["test tags", "testing"]
+            "author": "David Beazley"
         }
+
         response = self.client.patch(url, data, format='json')
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(response.data['description'], data['description'])
-        self.assertEqual(response.data['tags'][0]["name"], data['tags'][0])
```

Comment:
What happened to the assertions about tags here? (not necessarily wrong to bin them - its just not explained anywhere, I don't think)

Reply:
Yup.  That's due to #125 😄 .  I didn't even think to do a skip -- I'll add that in, I like how you did it with the issue ref in [49eca65](https://github.com/codebuddies/backend/commit/49eca650bbc4b1ec152201a7783bb5f32c349cf5) .

Reply:
as for binning the tags..I was solving an issue with the ordering being different coming back in `data` .. and then decided I didn't want to.  But I should do at least one test case with tags, so I can add it back in.